### PR TITLE
Site: update community page

### PR DESCRIPTION
**Pull request by prasadkatti**
Updated community page. The information about minikube meetings has been added to the "Develop and Contribute" section.

Before: 
https://minikube.sigs.k8s.io/community/

After: 
https://deploy-preview-7799--kubernetes-sigs-minikube.netlify.app/community/

### CODE REVIEW

**Comment by prasadkatti** [comment]
Hi @medyagh . I have added the required content. But some changes are in the main config.toml file. So it is not visible on Netlify PR link. But I have checked it by building the site locally and it works 👌 .

**Comment by prasadkatti** [comment]
Ohh actually, I updated the docsy theme partial. But that is a submodule, so I am not sure what is the right to make changes to a partial in a docsy (w/o upstreaming those changes).

**Comment by prasadkatti** [comment]
@medyagh - This is ready for another review.

File: site/layouts/partials/community_links.html [hunk 1]
```diff
@@ -10,11 +10,16 @@ <h2>Learn and Connect</h2>
...
 <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
 <h2>Develop and Contribute</h2>
+<p>minikube is a Kubernetes <a href="https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle">#sig-cluster-lifecycle</a> project.</p>
 <p>If you want to get more involved by contributing to {{ .Site.Title }}, join us here:
 {{ with index $links "developer"}}
 {{ template "community-links-list"  . }}
 {{ end }}
-<p>You can find out how to contribute to these docs in our <a href="/contributing/guide">Contributing Guide</a>.
+<p>You can find out how to contribute to these docs in our <a href="/docs/contribution-guidelines/">Contribution Guidelines</a>.
+<p>Join our meetings: </p>
```
Comment (medyagh) [inline site/layouts/partials/community_links.html:19]: make this as big as <h2>Develop and Contribute</h2>

Comment (prasadkatti) [reply site/layouts/partials/community_links.html]: Done.

**Pull request merged**